TestLoginTimeout is not reliable on low latency connections

AppVeyor test runs fail intermittently because TestLoginTimeout gets the wrong error type

```
--- FAIL: TestLoginTimeout (0.00s)
    queries_test.go:2047: Dialing host localhost with timeout 1ms
    queries_test.go:2097: Got error type *errors.errorString: unable to open tcp connection with host 'localhost:1433': dial tcp 127.0.0.1:1433: i/o timeout 
    queries_test.go:2103: wrong kind of error for login or query timeout: unable to open tcp connection with host 'localhost:1433': dial tcp 127.0.0.1:1433: i/o timeout
2022/08/19 14:02:55 [{Hello} {World} {TVP}]
FAIL
coverage: 79.2% of statements
exit status 1
FAIL	github.com/microsoft/go-mssqldb	25.273s
Command exited with code 1
```

We can make the test more reliable by looking for `timeout` in the error message.

--- a/queries_test.go
+++ b/queries_test.go
@@ -2099,7 +2099,8 @@ func TestLoginTimeout(t *testing.T) {
 		if !oe.Timeout() {
 			t.Fatalf("Got non-timeout error %s", oe.Error())
 		}
-	} else if err != context.DeadlineExceeded {
+		// The type of error is not guaranteed so just look for "timeout"
+	} else if err != context.DeadlineExceeded && !strings.Contains(err.Error(), "timeout") {
 		t.Fatalf("wrong kind of error for login or query timeout: %+v", err)
 	}
 

